README.md

```diff
@@ -87,6 +87,16 @@ Initialize the Datadog tracer when set to `true`. Defaults to `false`.
 
 Set to `true` to merge the X-Ray trace and the Datadog trace, when using both the X-Ray and Datadog tracing. Defaults to `false`.
 
+### DD_INFERRED_SPANS (beta)
```

Comment:
I don't know if you were planning to update this later, but when we're GA this should default to true.

We should also flag as "experimental" instead of "beta".